mission/config/ui/taskroster/main/ui.hpp

```diff
@@ -0,0 +1,229 @@
+//included by "ui_tr_base.hpp"
+
+class vn_tr_disp_taskRoster_main_rhs : vn_mf_RscControlsGroupNoScrollbarHV
```

Comment:
Every "page" in the task roster now has a Right Hand Side (RHS) config and a Left Hand Side (LHS) config.

The LHS is technically the both pages and we just add the RHS to it.